src/tests/test_certstor_macos.cpp

```diff
@@ -0,0 +1,336 @@
+/*
+* (C) 1999-2019 Jack Lloyd
+* (C) 2019      René Meusel
+*
+* Botan is released under the Simplified BSD License (see license.txt)
+*/
+
+#include "tests.h"
+
+#include <botan/build.h>
```

Comment:
You can assume "tests.h" includes `botan/build.h` since most tests want to do feature macro checks.